dask_planner/src/sql/types.rs

```diff
@@ -111,6 +111,29 @@ impl DaskTypeMap {
                 };
                 DataType::Timestamp(unit, tz)
             }
+            SqlTypeName::DECIMAL => {
+                let (precision, scale) = match py_kwargs {
+                    Some(dict) => {
```

Comment:
More of a nit, is there any way to enforce the kwargs we receive here in the case that additional kwargs beyond `precision` and `scale` are provided?